migrations/sqls/20241107093135-add-quarterly-returns-to-return-versions-up.sql

```diff
@@ -0,0 +1,2 @@
+alter table water.return_versions
+    add column quarterly_returns boolean default false;
```

Comment:
As a boolean, unless there is an explicit need IMHO it is also good practise to make it `NOT NULL`.

```suggestion
    add column quarterly_returns boolean not null default false;
```